TYPO3  backend login (#230)
TYPO3-Documentation/TYPO3CMS-Guide-Installation: Documentation/In-depth/TheInstallTool/Index.rst

impression of what you can configure. Normally you *can*, but you *don't
have to* change anything here during installation, as the previous steps
took care of the most important settings.

> Pay attention to use a https website and set the BE loginSecurityLevel to normal.

Reviewer: ```suggestion
Pay attention to use HTTPS for your website and set :code:`$GLOBALS['TYPO3_CONF_VARS']['BE']['loginSecurityLevel']` to `normal`.
```

This is not exactly true. Even with HTTPS you can use RSAAuth, it's just pointless. Thus we should rephrase this to a recommendation.

> On all http sites however you must set this to rsa and 

Reviewer: ```suggestion
However, on all websites without HTTPS you should set this to `rsa` and 
```

> activate the system extension rsaauth. Otherwise you will not be able to log

Reviewer: ```suggestion
activate the system extension `rsaauth`. Otherwise you will not be able to log
```

> activate the system extension `rsaauth`. Otherwise you will not be able to log

Reviewer 2: Should we also mention that using that this encryption approach is deprecated and is not part of the TYPO3 core anymore? https://github.com/FriendsOfTYPO3/rsaauth#current-state

Reviewer: Yes, good idea.

Reviewer 3: I wouldn't add that in the 8.7 docs- that's part of the upgrade process I think.

Reviewer: Indeed, I forgot the v8 context. This indeed only applies to v9+.
